doc/rst/platforms/udp.rst

```diff
@@ -161,5 +163,12 @@ You need to set ``CHPL_RT_MASTERIP`` (or ``GASNET_MASTERIP``), and possibly
   * ``$CHPL_HOME/third-party/gasnet/gasnet-src/udp-conduit/README``
   * https://gasnet.lbl.gov/dist/udp-conduit/README .
 
+For example, when simulating multiple locales by oversubscribing
+the local machine, use:
+
+.. code-block:: bash
+
+  export GASNET_MASTERIP=127.0.0.1
+  export GASNET_WORKERIP=127.0.0.0  # may be optional
```

Comment:
These particular values are really only needed when you're on a VPN that screws up gethostbyname. I'm not sure this makes sense in a user-facing doc. At the very least, let's change these to use the chapel env vars, and not gasnet.